doc/user/standalone-distribution.md

```diff
@@ -72,7 +72,7 @@ Set `TRUFFLERUBY_VERSION` to the latest TruffleRuby version from
 
```

Comment:
Changing the link above to https://github.com/oracle/truffleruby/releases/latest might be less confusing, could you maybe change the PR to do that?

Reply:
Sure! 👍